sqlx-mysql/src/protocol/statement/execute.rs

```diff
@@ -19,7 +20,7 @@ impl<'q> Encode<'_, Capabilities> for Execute<'q> {
         buf.extend(&1_u32.to_le_bytes()); // iterations (always 1): int<4>
 
         if !self.arguments.types.is_empty() {
-            buf.extend(&*self.arguments.null_bitmap);
+            buf.extend(&*self.arguments.null_bitmap.deref());
```

Comment:
It's not idiomatic to call `deref` directly. If you use `buf.extend_from_slice()` it should auto-deref.

Reply:
fixed!